utils/build/docker/python_otel/flask-poc-otel/app.py

```diff
@@ -0,0 +1,40 @@
+import os
+import random
+import subprocess
+
+import psycopg2
+import requests
+
+from flask import Flask, Response, jsonify
+from flask import request
+from flask import request as flask_request
+
+from integrations.db.mssql import executeMssqlOperation
+from integrations.db.mysqldb import executeMysqlOperation
+from integrations.db.postgres import executePostgresOperation
+
+app = Flask(__name__)
+
+
+@app.route("/")
+def hello_world():
+    return "Hello, World!\\n"
+
+
+@app.route("/db", methods=["GET", "POST", "OPTIONS"])
+def db():
+    service = flask_request.args.get("service")
+    operation = flask_request.args.get("operation")
+
+    print("REQUEST RECEIVED!")
```

Comment:
nit: Are we running the flask app in debug_mode? If so we should already get debug logs when a request is received.

We  should either remove this print statement or make it more useful. Maybe add the endpoint and the service/operation name that was received.

Reply:
I changed the trace